assessment-api/open-api-specification.yml

```diff
@@ -62,4 +130,79 @@ components:
           type: 'string'
           format: 'date'
         last_saved_by:
-          type: 'string'
\ No newline at end of file
+          type: 'string'
+    assessmentEntityDetail:
+      type: object
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+        attributes:
+          type: array
+          default: [ ]
+          items:
+            $ref: '#/components/schemas/assessmentAttributeDetail'
+        relations:
+          type: array
+          default: [ ]
+          items:
+            $ref: '#/components/schemas/assessmentRelationshipDetail'
+        prepopulated:
+          type: boolean
+      required:
+        - id
+        - entityId
+        - prepopulated
+    assessmentEntityTypeDetail:
```

Comment:
I know it makes no difference really, but if you move assessmentEntityTypeDetail above assessmentEntityDetail then the definitions are in the right order.